curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-082.md

```diff
@@ -8,7 +8,9 @@ isHidden: true
 ## Description
 <section id='description'>
 
-Now apply the `address` class to the `p` element containing the address.
+You change properties of a link when the link has actually been visited by using a <dfn>pseudo-selector</dfn> that looks like `a:visited { propertyName: propertyValue }`
```

Comment:
You change properties of a link when the link has actually been visited by using a <dfn>pseudo-selector</dfn> that looks like `a:visited { propertyName: propertyValue }`.